boards/arm/mimxrt1170_evk/mimxrt1170_evk_cm7.dts

```diff
@@ -32,6 +33,46 @@
 	};
 };
 
+&lcdif {
+	status = "okay";
+	width = <720>;
+	height = <1280>;
+	hsync = <8>;
+	hfp = <32>;
+	hbp = <32>;
+	vsync = <2>;
+	vfp = <16>;
+	vbp = <14>;
+	polarity = <3>;
+	pixel-format = "bgr-565";
+	data-buswidth = "24-bit";
+	backlight-gpios = <&gpio9 29 GPIO_ACTIVE_HIGH>;
+};
+
+&mipi_dsi {
+	status = "okay";
+	nxp,lcdif = <&lcdif>;
+	dpi-color-coding = "24-bit";
+	dpi-pixel-packet = "24-bit";
+	dpi-video-mode = "burst";
+	dpi-bllp-mode = "low-power";
+	autoinsert-eotp;
+	dphy-ref-frequency = <24000000>;
+
+	rm68200@0 {
+		status = "okay";
+		compatible = "raydium,rm68200";
+		reg = <0x0>;
+		label = "RM68200";
+		pwr-gpios = <&gpio11 16 GPIO_ACTIVE_HIGH>;
```

Comment:
Linux driver seems to use the regulator framework, see https://elixir.bootlin.com/linux/v5.17.5/source/drivers/gpu/drm/panel/panel-raydium-rm68200.c#L379. I'd personally do the same here. However, if the driver doesn't support PM you may just define a regulator that is always on in DT, then display doesn't even need to take care of turning it on.

Reply:
There are a few reasons. If the driver doesn't do any PM, the regulator can be configured to automatically be ON and so driver code doesn't even need to take care of managing display power. But it is not only that. Imagine there is a board where the display is part of a power rail that powers the display and something else. Then, the display can no longer manage the switch individually, so one would have to fork and modify the driver. This won't happen when using regulators, as they are reference counted.

Reply:
I have switched to using the regulator API to be automatically ON.